clients/admin-ui/src/features/organization/types.ts

```diff
@@ -1,9 +1,3 @@
-export interface Organization {
-  description: string;
-  fides_key: string;
-  name: string;
-}
-
 export interface OrganizationParams {
```

Comment:
@ssangervasi could use your guidance on if the other types in this file can be removed. I didn't find an equivalent one in the autogenerated types, but maybe it's under a sneaky name?

Reply:
Went looking for it too, and I think this might be left behind from some copy-paste. Listing organizations doesn't take any pagination params, so OrganizationParams isn't used. And `OrganizationResponse` is referenced, but incorrectly. The CRUD APIS return the object, not nested under a `response` property. The response is [ignored](https://github.com/ethyca/fides/blob/7693b175866683036731abaefc1ba0cab75e9737/clients/admin-ui/src/features/config-wizard/OrganizationInfoForm.tsx#L73) anyway, so the [slice](https://github.com/ethyca/fides/blob/7693b175866683036731abaefc1ba0cab75e9737/clients/admin-ui/src/features/organization/organization.slice.ts#L19) should just have the model type instead.